NewForce-Cohort-4/tabloid-full-stack-coral
# Jp create post

# Description

Allows the user to create a new post by inserting information about the post into a form. The form allows input for post title, content, image URL, and category. Upon submitting the post it is automatically approved and given a creation and published date and time. 

## Type of change


# How To Test

Start the app and login as a user. Navigate to "Posts" in the navbar then click the "Add New Post" button. Fill out the form and click "Submit." After submission you will be redirected to the details page of that post. Confirm all information has been added to the post. 

# Checklist:
